content/en/continuous_integration/tests/java.md

```diff
@@ -48,6 +48,17 @@ To report test results to Datadog, you need to configure the Datadog Java librar
 
 {{% ci-agentless %}}
 
+Additionally, configure [Datadog site][2] to which you want to send data.
+
+`DD_SITE` (Required)
+: The [Datadog site][2] to upload results to.<br/>
+**Default**: `datadoghq.com`<br/>
+**Selected site**: {{< region-param key="dd_site" code="true" >}}
+
+
+[1]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: /getting_started/site/
```

Comment:
I added this reusable shortcode that we can use instead: https://github.com/DataDog/documentation/blob/master/layouts/shortcodes/ci-agentless.md!

```suggestion
```